src/server/tablet_snap.rs

```diff
@@ -261,12 +267,12 @@ async fn recv_snap_files(
             };
             f.write_all(&chunk[..])?;
             limit.consume(chunk.len()).await;
+            SNAP_LIMIT_TRANSPORT_BYTES_COUNTER_STATIC
```

Comment:
why move here? need comments.  and size is useless?

Reply:
fixed